app/views/topics/show.html.erb

```diff
@@ -82,22 +82,20 @@
               <% end %>
             </div>
           <% end %>
-          <% if @topic.level_two? %>
```

Comment:
I don't think we can, as this would result in "Archive specialist topic page" link not showing on Level 1 topics so users wouldn’t be able to use the functionality this PR is adding.

At the same time, this PR doesn't allow to delete draft topics. Therefore we only want to show it for Level 2 topics. There will be follow up PR for this.

Reply:
Ahhh there's a restriction in [DraftTagRemover](https://github.com/alphagov/collections-publisher/blob/6e3069d4f9a5245dd9e2374ce5b1f8708393a37d/app/services/draft_tag_remover.rb#L10) remover that we can't delete level one tags. I hadn't spotted that. That makes sense, thanks 👍